contrib/namespace-type-lib/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@ory/keto-namespace-types",
+  "version": "0.9.0-alpha.0",
```

Comment:
Do we have to manage that version manually?

Reply:
No, the CI release does that :wink: https://github.com/ory/keto/pull/1029/files#diff-690379e7730ef01d6a0d83d5590312f76fbb839b4642554752d757b98e85365aR27